site/index.html

```diff
@@ -30,13 +30,13 @@
       </div>
 
       <div class="competition_title_block">
-        <h1>Rinse, Recycle, Repeat College Competition</h1>
+        <h1 >Rinse, Recycle, Repeat College Competition</h1>
```

Comment:
random extra space here, probably not affecting anything...

Reply:
oh I'll get rid of that now! Thanks for catching that ! 👀